presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -263,4 +262,29 @@ private String getProcessWorkingPath(String path)
 
         return absolutePath.getAbsolutePath();
     }
+
+    private List<String> getLaunchCommand() throws IOException
+    {
+        String executablePath = getProcessWorkingPath(SystemSessionProperties.getNativeExecutionExecutablePath(session));
+        String programArgs = SystemSessionProperties.getNativeExecutionProgramArguments(session);
+        String configPath = Paths.get(getProcessWorkingPath("./"), String.valueOf(port)).toAbsolutePath().toString();
+        ImmutableList.Builder<String> command = ImmutableList.builder();
+        List<String> argsList = Arrays.asList(programArgs.split(" "));
```

Comment:
should we use `programArgs.split("\\s+")` as it could be accidentally more than one spaces in the args list?

Reply:
It should be fine. It will be ignored. But \\s+ looks more legit.